packages/core/forms/src/components/fields/FieldMultiselect.vue

```diff
@@ -1,41 +1,89 @@
 <template>
   <KMultiselect
+    :id="getFieldID(schema)"
+    v-model="inputValue"
     :aria-labelledby="getLabelId(schema)"
+    :class="schema.fieldClasses"
     data-testid="field-multiselect"
+    :disabled="disabled || undefined"
```

Comment:
```suggestion
    :disabled="disabled"
```

I believe it's unnecessary to fallback to `undefined` here?